### ISSUE TITLE
license

### ISSUE THREAD

**Issue by jringstad** [gabgoh/epcalc#11]
Hey,
could you add a license? I'd love to embed this into some internal views etc

**Comment by AndrewJamesTurner** [comment]
Yes, I too am working on using this code base and would like an explicit license detailing the terms of use. If it helps I have included a link to commonly used opensource licenses you could choose: https://choosealicense.com/licenses/

**Comment by Scott-Gall** [comment]
Good Day,

I also would love to leverage this code for a project, could you please choose a license as AndrewJamesTurner suggests?

Thank you very kindly.

**Comment by leliel12** [comment]
+1

**Comment by dpaz2118** [comment]
+1
